Account/views.py

```diff
@@ -45,12 +45,12 @@ def profile(request, username):
             #action once they are logged in
             return HttpResponseRedirect(reverse('Account:login'))
 
-        if request.POST.get('follow' + profile_user.username):
+        if request.POST.get('follow'):
```

Comment:
You will see that I removed the reference of  `'follow' + profile username` to just `'follow'`.

This allows a simplified approach when working with the POST requests...since we have multiple different follow buttons on a given page (that follow different accounts) we do not want to create a separate if statement to handle each post request. 

So now one POST.get('follow') will handle all the follow POST requests. We differentiate which follow button was clicked by what the actual submitted value of the follow button is. Whenever a follow button is clicked it submits the users profile name that we can reference so we know who to follow.